src/util.c

```diff
@@ -40,6 +40,16 @@ bool strendswith_any(const char *s, const char **earray) {
     return false;
 }
 
+bool strstartswith_any(const char *s, const char **earray) {
```

Comment:
Good catch, but it's not misnamed, it's broken... strendswith_any just above it is exactly the same. Ctrl+C/Ctrl+V programming :/

It's not used anywhere yet, I just added it for parity. I probably shouldn't do that without testing the thing at least.